generators/app/templates/.woloxci/Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /home/node
 
 COPY package.json .
 
-RUN npm install
+RUN yarn install
```

Comment:
If we are going to use bitrise for CI in all of our projects, Don't we have to delete the `.woloxci` from this folder?

Reply:
yep, but thats another PR

Reply:
oka